sdks/common/runner-api/src/main/proto/beam_runner_api.proto

```diff
@@ -367,9 +367,12 @@ message WriteFilesPayload {
   // (Required) The SdkFunctionSpec of the FileBasedSink.
   SdkFunctionSpec sink = 1;
 
-  bool windowed_writes = 2;
+  // (Required) The format function.
```

Comment:
Can you describe more what the format function is? A runner has the following choices, really:

1. Do not try to do anything special.
2. Build something that can be invoked via the Fn API out of the format function
3. Just use the format function to build an SDK-specific derived transform
4. ???